File: CTDK_Python/100/_100.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 0
t = 1
i = 1
while(i <= n):
    t = t * i
    s = (t + s)**(1 / (i + 1))
    i = i + 1

print("Tong can la: ", s)

File: LTHAM/100/100.cpp
#include <iostream>
#include <cmath>
using namespace std;

float Tong(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	cout << "S = " << Tong(n) << endl;
	return 0;
}

float Tong(int n)
{
	float s = 0;
	int t = 1;
	int i = 1;
	while (i <= n)
	{
		t = t * i;
		s = pow(s + t, (float)1 / (i + 1));
		i++;
	}
	return s;
}